src/web/components/App.tsx

```diff
@@ -232,7 +233,7 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 			const hasOptedOut = await hasOptedOutOfArticleCount();
 			if (!hasOptedOut) {
 				incrementDailyArticleCount();
-				incrementWeeklyArticleCount();
+				incrementWeeklyArticleCount(storage.local, CAPI.pageId);
```

Comment:
I guess we should add `CAPI.pageId` to the list of dependencies for this `useEffect`?

Reply:
I suspect it wouldn't make a difference (CAPI.pageId shouldn't change), but I think `[]` makes it clear that this effect must only run once